CMakeLists.txt

```diff
@@ -482,33 +482,125 @@ find_package(Boost)
 if(Boost_FOUND AND BUILD_SHARED)
     include_directories(${Boost_INCLUDE_DIRS} ${Boost_INCLUDE_DIR})
     option(WITH_MAEPARSER "Build Maestro support" ON)
+    option(WITH_COORDGEN "Build Coordgen support" ON)
 else()
     option(WITH_MAEPARSER "Build Maestro support" OFF)
+    option(WITH_COORDGEN "Build Coordgen support" OFF)
 endif()
 
-
 if(WITH_MAEPARSER)
-    set (CMAKE_CXX_STANDARD 11)
-
-    set(MAEPARSER_VERSION 1.1)
-    if(NOT EXISTS "${openbabel_SOURCE_DIR}/external/maeparser-${MAEPARSER_VERSION}")
-        file(DOWNLOAD "https://github.com/schrodinger/maeparser/archive/v${MAEPARSER_VERSION}.tar.gz"
-            "${openbabel_SOURCE_DIR}/external/maeparser-${MAEPARSER_VERSION}.tar.gz" STATUS status)
-        execute_process(COMMAND ${CMAKE_COMMAND} -E tar zxf
-            ${openbabel_SOURCE_DIR}/external/maeparser-${MAEPARSER_VERSION}.tar.gz
-            WORKING_DIRECTORY ${openbabel_SOURCE_DIR}/external)
-        message(STATUS "Downloaded MaeParser to ${openbabel_SOURCE_DIR}/external/maeparser-${MAEPARSER_VERSION}.")
+
+    find_package(maeparser MODULE QUIET)
+
+    if (maeparser_FOUND)
+
+      message(STATUS "Maestro formats will be supported. Using MaeParser libraries at ${maeparser_LIBRARIES}.")
+
+    else()
+
+      set(MAEPARSER_VERSION "master")
```

Comment:
I'm a bit hesitant at that. I don't mind "let's use master for OB master".. is the API for maeparser fixed? After all, if maeparser changes, does the code here in OB need an update too?

Reply:
That can be changed, of course. In fact, I just noticed a couple of things I forgot about when I wrote that. I'll update shortly.